web-app/src/models/Subject.py

```diff
@@ -17,7 +17,16 @@ class Subject(db.Model):
         nullable=False
     )
 
-
+    short = db.Column(
+        db.String(20),
+        unique=True,
+        nullable=False
+    )
+    
+    tutor = db.relationship(
```

Comment:
Das kannst du einfach auf eine Referenz zur Users-Tabelle umbiegen. Also beispielsweise so:

`tutor_id = db.Column(
        db.Integer,
        db.ForeignKey('users.id'),
        nullable=False
    )`

Ich meine dass das auch falsch in IQA-83 beschrieben war mit dem Tutor.
Der Tutor existiert ja dann als User in der Datenbank und muss sich später auch anmelden können. Und dann haben wir weniger Redundanz.